store/src/scoring.rs

```diff
@@ -77,13 +77,39 @@ async fn voters_with_marinade_stake_in_epoch(
         .collect())
 }
 
+async fn voters_credits_performance_in_epoch(
+    psql_client: &Client,
+    epoch: u64,
+) -> anyhow::Result<HashMap<String, f64>> {
+    log::info!(
+        "Loading list of poor voters: {}",
+        epoch
+    );
+    Ok(psql_client
+        .query(
+            "WITH stats AS (SELECT AVG(activated_stake * credits) / avg(activated_stake) as stake_weighted_avg_credits FROM validators WHERE epoch = $1)
+            SELECT
+                vote_account,
+                credits,
+                coalesce(credits / stake_weighted_avg_credits, 0)::double precision as credits_performance
+            FROM validators left join stats on 1 = 1
+            WHERE epoch = $1",
+            &[&Decimal::from(epoch)],
+        )
+        .await?
+        .iter()
+        .map(|row| (row.get("vote_account"), row.get("credits_performance")))
+        .collect())
+}
+
 pub async fn load_unstake_hints(
     psql_client: &Client,
     blacklist_path: &String,
     epoch: u64,
 ) -> anyhow::Result<Vec<UnstakeHintRecord>> {
     log::info!("Loading unstake hints in epoch: {}", epoch);
     let max_allowed_commission = 10;
```

Comment:
```suggestion
    const MAX_ALLOWED_COMMISSION: u8 = 10;
```
I think it should be a constant and placed on top of the file for better visibility